addons/mail/static/src/discuss/core/web/thread_actions.js

```diff
@@ -15,14 +15,11 @@ threadActionsRegistry.add("expand-discuss", {
     name: _t("Open in Discuss"),
     open(component) {
         component.thread.setAsDiscussThread();
-        component.actionService.doAction(
-            {
-                type: "ir.actions.client",
-                tag: "mail.action_discuss",
-                name: _t("Discuss"),
-            },
-            { clearBreadcrumbs: true }
-        );
+        component.actionService.doAction({
+            type: "ir.actions.client",
+            tag: "mail.action_discuss",
+            name: _t("Discuss"),
```

Comment:
This should be the name of the thread rather than "discuss" otherwise it overrides the name from `setDisplayName` in `setAsDiscussThread` (it's called before).

The issue is when using expand from chat window, the title of the page/breadcrumb will be Discuss rather than the thread name.